### ISSUE TITLE
 Error There was an issue checking in the license. Please try again. 

### ISSUE THREAD

**Issue by haneesibm** [grokability/snipe-it#12601]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

no able to check-in any license assigned to users
this is started after upgrading the new master version of snipeit v6.0.14 - build 9646 (master)

### Reproduction steps

1. select the license and check-in from the user its been assigned. 
2.
3.
...

### Expected behavior

check in the license from users. 

### Screenshots

![image](https://user-images.githubusercontent.com/35599140/222605598-d2aeda1b-384d-484a-b345-05384b616830.png)

![image](https://user-images.githubusercontent.com/35599140/222608387-0be891be-acf0-4970-9d57-813e7d42db26.png)

### Snipe-IT Version

v6.0.14 - build 9646 (master)

### Operating System

Windows

### Web Server

IIS

### PHP Version

8.0.28

### Operating System

Windows 10

### Browser

Google Chrome & Mozilla Firefox

### Version

110

### Error messages

```shell
$ php -m
[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
fileinfo
filter
gd
gettext
hash
iconv
imap
json
ldap
libxml
mbstring
mysqli
mysqlnd
openssl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
readline
Reflection
session
SimpleXML
soap
sodium
SPL
standard
tidy
tokenizer
wincache
xml
xmlreader
xmlrpc
xmlwriter
zip
zlib
[Zend Modules]
```

### Additional context

its an upgrade of new version. 
i have able to reproduce the same error in  https://snipeitapp.com/demo. master Demo.

**Comment by inietov** [comment]
Thanks for your report! I just modified this behaviour, let me investigate.

**Comment by haneesibm** [comment]
the issue is resolved by fixing the variable as suggested in #12602
